### DISCUSSION TITLE
How do I set my favourite browser in v16?

### DISCUSSION THREAD

**Discussion by will-stone** [will-stone/browserosaurus#474] [Q&A] [answered]
How do I set my favourite browser in v16?

**Comment by will-stone** [comment, marked as answer]
A favourite browser will show a larger icon than the rest of the apps. It will also move to the front of the picker window, and is therefore an easier target to click with the mouse. You can set your favourite in the Preferences menu (found in the "B" menubar icon), by dragging your favoured app to the top of the Apps pane.

**Comment by xiaolianggit** [comment]
is this no longer supported?

I'm on 18.4.2 and I don't see the option to set a favorite.

**Reply by will-stone** [reply]
Nope. No need for a favourite, but you can think of it as the one at the top ☺️
